components/Feature.tsx

```diff
@@ -0,0 +1,44 @@
+"use client"
+import { cn } from "@/lib/utils"
+import { motion } from "framer-motion"
+import { Button } from "./ui/button"
+
+type FeatureCardProps = {
+  heading: string
+  body: string
+  reverse?: boolean
+}
+
+export default function Feature({
+  heading,
+  body,
+  reverse,
+  ...props
```

Comment:
co innego tutaj moze byc?

Reply:
myślałem żeby zostawić na przyszłość ale lepiej to usunąć xd